pmoves/services/p7-room-orchestrator/app.py

```
@@ -0,0 +1,331 @@
+
+class NATSPublisher:
+    def __init__(self):
+        self._nc = None
+
+    async def connect(self):
+        try:
+            from nats.aio.client import Client as NATSClient
+            self._nc = NATSClient()
+            await self._nc.connect(servers=[NATS_URL])
+            LOG.info("Connected to NATS at %s", NATS_URL)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)